src/main/kotlin/com/jetbrains/micropython/settings/MicroPythonFacet.kt

```diff
@@ -61,10 +61,13 @@ class MicroPythonFacet(facetType: FacetType<out Facet<*>, *>, module: Module, na
   }
 
   override fun updateLibrary() {
-    val typeHints = configuration.deviceProvider.typeHints
     val plugin = getPluginDescriptor()
-    val paths = if (typeHints != null) listOf("${plugin.path}/typehints/${typeHints.path}") else emptyList()
-    FacetLibraryConfigurator.attachPythonLibrary(module, null, "MicroPython", paths)
+
+    val boardTypeHints = configuration.deviceProvider.typeHints
+    val commonHintsPaths = listOf("${plugin.path}/typehints/stdlib", "${plugin.path}/typehints/micropython")
```

Comment:
There are no stdlib and MicroPython modules available for BBC Micro:bit. Let's put all the paths to device providers. Probably, by modifying `MicroPythonTypeHints` to contain `paths: List<String>`.

In near future I will merge all ESP providers into the single ESP provider that will contain its three paths.